integration_tests/e2e/workAndSkillsPage/workAndSkillsGoals.cy.ts

```diff
@@ -103,4 +106,60 @@ context('Work and skills page - Goals card', () => {
     workAndSkillsPage.GoalsLongTermText().should('exist')
     workAndSkillsPage.GoalsLongTermText().contains('The prisoner does not have any long term goals.')
   })
+
+  it.skip('should display the Goals card with populated goals given prisoner has PLP goals only', () => {
```

Comment:
These are the new scenarios, all ready and waiting for their assertions to be fleshed out 😁